main/consts.js

```diff
@@ -21,6 +21,10 @@ module.exports = Object.freeze({
 // Replace with `app.get('localUserData')` after this PR is landed & released:
 // https://github.com/electron/electron/pull/34337
 function getCacheHome () {
+  if (process.env.NODE_ENV === 'test' && process.env.HOME) {
+    return path.join(process.env.HOME, 'local-user-data')
```

Comment:
Yeah, this isn't very clear.

The thing is, `process.env.HOME` is set to a random temp dir by our E2E test suite; see here:
https://github.com/filecoin-station/filecoin-station/blob/5e92791706195ccb5adff58983eb66510b4baa18/test/e2e/app-launch.e2e.test.js#L27-L36

I am thinking about the following solution to make thinks easier to understand:
1. In the E2E test setup, define a new env variable for the random temp dir, e.g. `TEST_ROOT_DIR`.
2. Modify the code I am adding in this PR to use `TEST_ROOT_DIR` instead of `HOME`.

What do you think? Can you suggest a better name for the new env var? Maybe `TEMP_ROOT_DIR` would better convey the information about the uniqueness of the directory.

Reply:
If overwriting `$HOME` isn't the right way to solve this, maybe we can remove it and only pass `STATION_ROOT` or so, which is an optional environment variable

Reply:
> I think it would be tricky to ensure we always remember to check for `STATION_ROOT` whenever we are accessing a new `app.getPath()` location.

On the second thought, we need `STATION_ROOT` to work reliably to ensure our tests are running isolated from the "main" station config, so we need the implementation to be 100% correct anyways.

I am going to use `STATION_ROOT`.

Reply:
>  I think $HOME does control certain locations on certain platforms. Typically on Linux, many paths are defaulting to user HOME directory.

As long as we aren't aware of anything inside Station doing this, I believe it will be clearer to remove it

> or should we implement it as a public (undocumented?) feature?

Desktop apps don't really respect environment variables, so I wouldn't say this is a user facing feature